### DISCUSSION TITLE
How can I install an older version?

### DISCUSSION THREAD

**Discussion by ivaquero** [Homebrew/discussions#1734] [Getting started] [answered]
Hi, how can I install suite-sparse-5.4.0  instead of the current suite-sparse-5.1.0

**Comment by carlocab** [comment, marked as answer]
You'll need to `brew extract` it into your own tap. For example:
```
brew tap-new integzz/insert-tap-preferred-name-here
brew extract --version=5.4.0 suite-sparse integzz/insert-preferred-tap-name-here
```
This will create a `suite-sparse@5.4.0.rb` in `$(brew --repo integzz/insert-preferred-tap-name-here)/Formula`. You can edit it as you please, and then do
```
brew install integzz/insert-preferred-tap-name-here/suite-sparse@5.4.0
```
This may not build as is, so you'll need to adjust the formula accordingly.

Here are some other docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Reply by ivaquero** [reply]
Thank you.
